bin/utils/ObjUtils.ts

```diff
@@ -49,6 +49,7 @@ export class Flattener {
 
     flatten1(json:any, first:boolean, keyPrefix:string, result:any):void {
         if (Array.isArray(json)){
+            result[keyPrefix] = json.join(',');
```

Comment:
can i make this into a conditional as part of the arg? because not everyone may want this feature...
this is the feature where an entire array will not only be seen as a0, a1, but allso that a=0,1